build.bat

```diff
@@ -367,6 +383,16 @@ if !BUILD_CPP! == 1 (
 	set "additional_header_files=!additional_header_files_custom_type!!additional_header_files!RTIRawTransportImpl.h Parameter.h ParameterManager.h ThreadPriorities.h RTIDDSLoggerDevice.h MessagingIF.h RTIDDSImpl.h perftest_cpp.h qos_string.h CpuMonitor.h PerftestTransport.h Infrastructure_common.h Infrastructure_pro.h"
 	set "additional_source_files=!additional_source_files_custom_type!!additional_source_files!RTIRawTransportImpl.cxx Parameter.cxx ParameterManager.cxx ThreadPriorities.cxx RTIDDSLoggerDevice.cxx RTIDDSImpl.cxx CpuMonitor.cxx PerftestTransport.cxx Infrastructure_common.cxx Infrastructure_pro.cxx"
 
+	@REM # TODO: REMOVE THIS
```

Comment:
We cannot have a comment like this in the code. I believe we decided that for supporting codegen for 6.0.0 we need this, right? This is the same we are doing in the .sh

Reply:
I belive we only need to do it twice right? not 3 times.

Reply:
I forgot to remove that. I was printing the command to be executed to debug it. But now that we know it works I will remove it.